src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -218,26 +217,28 @@ export const VolunteerFormCalendar = ({
         {!!groupId || isEditMode ? (
           <Input hidden name="contentContainerId" control={control} />
         ) : (
-          <Controller
-            name="contentContainerId"
-            render={({ name, onChange, value }) => (
-              <DropdownInput
-                {...{
-                  errors,
-                  required: true,
-                  data: groupDropdownData,
-                  value,
-                  valueKey: 'contentcontainer_id',
-                  onChange,
-                  name,
-                  label: texts.volunteer.group,
-                  placeholder: texts.volunteer.group,
-                  control
-                }}
-              />
-            )}
-            control={control}
-          />
+          groupDropdownData?.length && (
```

Comment:
Line 188 has an if check. If it passes that if, won't we already have `groupDropdownData?.length`? Do you think this check is necessary? 

Do you need the question mark in the `groupDropdownData?.length` control? Can't we just check `groupDropdownData.length`?

Reply:
yes i think we need it because its initial state is `undefined`. line 188 checks in combination with `isSuccessGroupsMy`. so if there is success of the query we do not know that `groupDropdownData` is an array or undefined, or am i wrong?

Reply:
That's right. Let's keep it that way.